src/streamIO.hpp

```diff
@@ -354,8 +353,9 @@ std::ostream & operator<<( std::ostream & stream,
  * @param view The SortedArrayView to output.
  * @return @p stream .
  */
-template< typename T, typename INDEX_TYPE >
-std::ostream & operator<< ( std::ostream & stream, SortedArrayView< T const, INDEX_TYPE > const & view )
+template< typename T, typename INDEX_TYPE, template< typename > class BUFFER_TYPE >
+std::ostream & operator<< ( std::ostream & stream,
```

Comment:
in addition to these stream operators, it would be nice to have an output option that would dump using `printf` so that we could use it on device. For instance a `printRow( INDEX_TYPE const row )`.

Reply:
#165